src/main/java/com/lambdaschool/oktafoundation/models/PartnerApplication.java

```diff
@@ -1,6 +1,7 @@
 package com.lambdaschool.oktafoundation.models;
 
 import javax.persistence.*;
+import java.util.Date;
```

Comment:
Random java quirk. `java.util.Date` is actually pretty terrible. I unknowingly tried to use it in a previous project and ran into tons of problems. It worked fine on the backend. But getting a date object from JSON or any sort of string was impossible.

Can we convert this, and all the others to `java.time.LocalDateTime` instead? That should work just fine with the JS's Date() object.

See [this article](https://programminghints.com/2017/05/still-using-java-util-date-dont/) for more details